### ISSUE TITLE
IIS on W2K8 ?

### ISSUE THREAD

**Issue by anahimself** [unosquare/passcore#55]
Does this work out of the box with IIS running on W2K8?
I am afraid the handler isnt compatible with IIS7+ but only IIS8+?

**Comment by coltonoslund** [comment]
I am wondering the same?

**Comment by anahimself** [comment]
I used APACHE + PHP finally (unfortunately)..
A little more coding but good examples out there..

**Comment by dsghi** [comment]
I was able to get it working without much trouble on Server 2008 R2 (IIS 7.5).

The special steps I took that deviated were installing the .NET Core Windows Server Hosting bundle using steps 1 & 2 in this document: https://docs.microsoft.com/en-us/aspnet/core/publishing/iis#install-the-net-core-windows-server-hosting-bundle.

After that I basically followed the steps in the main instructions, the only difference was that "AlwaysRunning" isn't an option for the App Pool, but the timeout value is so that should be sufficient.

To paraphrase:
&nbsp;&nbsp;&nbsp;&nbsp;Step 1 : Download and install: https://go.microsoft.com/fwlink/?linkid=848766
&nbsp;&nbsp;&nbsp;&nbsp;Step 2 : Reboot
&nbsp;&nbsp;&nbsp;&nbsp;Step 3 : Create App Pool
&nbsp;&nbsp;&nbsp;&nbsp;Step 4 : Create Website
&nbsp;&nbsp;&nbsp;&nbsp;Step 5 : Adjust config file and enjoy

(P.S. If you end up with a blank webpage that just keeps loading, make sure you "unblocked" the zip file before you extracted it. This caught me twice, lol)
